translator/translate/otel/exporter/awsxray/translator.go

```diff
@@ -32,6 +32,12 @@ type translator struct {
 
 var _ common.Translator[component.Config] = (*translator)(nil)
 
+var indexedAttributes = []string{
```

Comment:
if we are moving this to global, we should specify this is appsignals related to not confuse this with default xray functionality.

Reply:
Would it be better to just have a list of index attributes for EKS and K8S as separate variables? Makes it more modular if we want to change it in the future.